source/index.txt

```diff
@@ -1,10 +1,51 @@
+:noprevnext:
+
 =========================
 {+atlas-arch-center+}
 =========================
 
 .. default-domain:: mongodb
 
-Landing page content
+The {+atlas-arch-center+} documents the best way to build a robust data
+platform in {+service+}. While the :atlas:`{+service+} documentation 
+</>` provides information about how to use {+service+}, the 
+{+atlas-arch-center+} gives recommendations on the exact 
+configurations you should use to optimize governance, security,
+automation, and more.
+
+The {+atlas-arch-center+}'s content aligns with the {+waf+} pillars:
+
+.. image:: /includes/images/landing-image.png
```

Comment:
Issue/Question: So I typically use dark mode, and in dark mode, the Operational Efficiency and Cost Optimization labels aren't readable. Are there plans to add a dark mode friendly image too?

Reply:
Great catch, I hadn't check dark mode. Let me ask DOP.

Reply:
Updated with dark mode compliant image! Thank you!